src/components/topic/TopicDirective.js

```diff
@@ -31,6 +31,21 @@ goog.require('ga_topic_service');
               scope.activeTopic = newTopic;
             };
 
+            scope.localeSensitiveComparator = function(o1, o2) {
+              var v1 = o1.value;
+              var v2 = o2.value;
+              var groupId1 = v1.groupId;
+              var groupId2 = v2.groupId;
+              if (groupId1 != groupId2 && typeof(groupId1) == 'number' &&
+                  typeof(groupId2) == 'number') {
```

Comment:
Yes it is, I was thinking that it was better if optional, that was the behaviour before and wanted to make it compapatible geoadmin forks.